.github/pull_request_template.md

```diff
@@ -1,30 +1,25 @@
-## Description & motivation
+## What are you changing in this pull request and why?
 <!---
-Describe your changes, and why you're making them. Is this linked to an open
-issue, a pull request on dbt core, etc?
+Describe your changes, and why you're making them. If linked to an open
+issue or a pull request on dbt core, then link to them here! 
 
 To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
 -->
 
-## To-do before merge
-<!---
-(Optional -- remove this section if not needed)
-Include any notes about things that need to happen before this PR is merged, e.g.:
-- [ ] Change the base branch
-- [ ] Ensure PR #56 is merged
+## Checklist
+<!--
+Uncomment if you're publishing docs for a prerelease version of dbt (delete if not applicable):
+- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
```

Comment:
Do we have a list of "must review" pages whenever a new version is released? If not, I can put something together.

Reply:
@matthewshaver I started a release checklist here: https://github.com/dbt-labs/docs.getdbt.com/pull/1355 I wonder if it would work better as its own doc in the Contributing folder?